cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaMirrorMaker2ClusterTest.java

```diff
@@ -1823,7 +1876,7 @@ public void testPodSetWithOAuthWithTls() {
                 .build();
         KafkaMirrorMaker2 resource = new KafkaMirrorMaker2Builder(this.resource)
                 .editSpec()
-                    .withClusters(targetClusterWithOAuthWithTls)                
+                .withClusters(targetClusterWithOAuthWithTls)
```

Comment:
This indentation and the indentations above are intentional to make it easier to read the builders. So please keep them as they were (here, above this line ... as well as possibly in any other files).